README.md

```diff
@@ -116,4 +116,4 @@ Apache 2.0 - See [LICENSE] for more information.
 [cloud-platform]: https://cloud.google.com/
 [product-overview]: https://developers.google.com/games/services/
 [product-docs]: https://developers.google.com/games/services/
-[lib-docs]: https://googleapis.dev/java/google-cloud-clients/latest/index.html?com/google/cloud/gameservices/v1alpha/package-summary.html
\ No newline at end of file
+[lib-docs]: https://googleapis.dev/java/google-cloud-clients/latest/index.html?com/google/cloud/gameservices/v1alpha/package-summary.html
```

Comment:
what's the change here?

Reply:
I didn't edit that line. Perhaps github added a \n at the eof or some such?